service/create/security_group.go

```diff
@@ -113,14 +117,14 @@ func (ri rulesInput) workerRules() []awsresources.Rule {
 		},
 		{
 			Port:       calicoBGPNetworkPort,
-			SourceCIDR: defaultCIDR,
+			SourceCIDR: ri.MastersSecurityGroupID,
```

Comment:
Why is an ID applied to a CIDR value? I feel like I miss something. IMO this should be assigned as `SecurityGroupID`.